packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -894,6 +898,8 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin,
       _showHideCursor();
       _showCursor.addListener(_showHideCursor);
     }
+    _internalShowCursor?.dispose();
```

Comment:
The only goal of this field is to dispose cursor.
So, the name `_internalShowCursor` does not describe the meaning.
Also, as it is separate from `_showCursor` it is not very clear when the field should be disposed.

So, I still suggest to have a flag like `_disposeShowCursor`, and then it is clear that current value of _showCursor should be disposed if it is not needed any more.

How does it sound?

Reply:
@polina-c, thanks for the review!
I totally agree with you, `_disposeShowCursor` is self-explanatory.